backend/public-graph/graph/resolver.go

```diff
@@ -553,7 +554,12 @@ func (r *Resolver) HandleErrorAndGroup(ctx context.Context, errorObj *model.Erro
 		return nil, errors.New("error object stacktrace was empty")
 	}
 
-	if projectID == 1 {
+	project, err := r.Store.GetProject(projectID)
```

Comment:
We are now doing an extra lookup for the project which adds one query to this code path. However, the trade off does seem worth it as we are ensuring errors are only being processed for valid projects (i.e. garbage project ids can't be thrown in). See the tests that needed to be updated to support this.